### ISSUE TITLE
Jobs running with errors using Log Execution processor results in unhelpful output

### ISSUE THREAD

**Issue by debunn** [victorcoder/dkron#484]
## Expected Behavior
When using the Log Execution Processor, jobs that return an error should send stdout messages to stdout, and stderr messages to stderr

## Actual Behavior
An unhelpful "rpc error: code = Unknown desc = exit status 1" type of message is output

## Steps to Reproduce the Problem

  1. Create a test batch job on the dkron client (/scripts/test.bash):
  ```
  #!/bin/bash
  echo "This is a test script"
  rm /file/does/not/exist
  ```
  2.  Create a test job on the client / server:
  ```
  {
	"name":"test",
	"executor": "shell",
	"executor_config": {
		"command": "/scripts/test.bash",
		"shell": "true"
	},
	"schedule":"@every 2m",
	"concurrency": "forbid",
	"tags":{
		"role":"job"
	},
    "processors": {
        "log": {
            "forward": true
        }
    }
}
  ```
  3. Watch the stdout / stderr of the server for the output of the job:  (stdout) "This is a test script", (stderr) "rm: cannot remove '/file/does/not/exist': No such file or directory".  Neither of these outputs are reported to either the client or server - instead a cryptic "rpc error: code = Unknown desc = exit status 1" message is sent instead

## Specifications
  - Version: 1.0.2
  - Platform: dkron server (Docker - Alpine Linux 3.8.2), dkron client (CentOS 7.6)
  - Backend store: Consul 1.0.6

Note:  The cryptic "rpc error: code = Unknown desc = exit status 1" error message is also concatenated to the same line as the "----- END OUTPUT -----" text - it might be a good idea to include a "\n" command at the end of the stderr output to ensure these are on separate lines.

**Comment by fopina** [comment]
Struggling with this as well and I'd say it narrows down to this:

https://github.com/victorcoder/dkron/blob/master/builtin/bins/dkron-executor-shell/shell.go#L90

exit code != 0 results in err, so it return `nil` as output, regardless of actual output...

**Comment by fopina** [comment]
For now, I'm using this small workaround https://github.com/fopina/dkron/commit/b69ad8d90d31e365a77ec2e87199fb687fa200c1#diff-53f30a09e9285d217e25127f0dee413f concatenating error with output...
Tried to pass output along with err, changed 3 or 4 different lines but it still gets lost somewhere along the whole grpc path...

**Comment by victorcoder** [comment]
Related to https://github.com/victorcoder/dkron/pull/448

**Comment by fopina** [comment]
Nice, should have dug a little more though if all that is required, I guess I’m happy with my workaround (concatenate output with error) in the meantime :)

**Comment by fopina** [comment]
@victorcoder #448 looks a bit stale, would it be worth opening a new PR to achieve the same trying to break as little as possible or you're not interested if there are any breaking changes? (I think proto will always have to be modified to support 2 separate properties)

**Comment by victorcoder** [comment]
@fopina I hope it only lacks my review and merge, I'm inclined to go with it.

**Comment by victorcoder** [comment]
Fixed in #448
